<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceWidget</class>
 <widget class="QWidget" name="FindReplaceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>Find:</string>
     </property>
     <property name="buddy">
      <cstring>findLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QToolButton" name="findOptionToolButton">
       <property name="text">
        <string>...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findLineEdit">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="findButtonsWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="findPreviousButton">
        <property name="text">
         <string/>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="findNextButton">
        <property name="text">
         <string/>
        </property>
        <property name="arrowType">
         <enum>Qt::RightArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="closeButton">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../resources.qrc">
          <normaloff>:/icons/icon-close.png</normaloff>:/icons/icon-close.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="replaceLabel">
     <property name="text">
      <string>Replace with:</string>
     </property>
     <property name="buddy">
      <cstring>replaceLineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="replaceLineEdit">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QWidget" name="replaceButtonsWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="replaceButtonsLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="replaceButton">
        <property name="text">
         <string>Replace</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
        <property name="arrowType">
         <enum>Qt::LeftArrow</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="replaceAllButton">
        <property name="text">
         <string>Replace All</string>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextOnly</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>findNextButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceWidget</receiver>
   <slot>findNextClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>427</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findPreviousButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceWidget</receiver>
   <slot>findPreviousClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>34</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceWidget</receiver>
   <slot>replaceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>429</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceAllButton</sender>
   <signal>clicked()</signal>
   <receiver>FindReplaceWidget</receiver>
   <slot>replaceAllClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>findLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>findNextButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>23</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>23</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
